conf/provisioning/alertrules/rules2.yml

```diff
@@ -0,0 +1,56 @@
+apiVersion: 1
+groups:
+  - name: my_group
+    folder: my_folder
+    interval: 10s
+    rules:
+    - title: my_second_file
+      uid: my_second_file
+      condition: B
+      for: 5m
+      annotations:
+        runbook: http://google.com/
+      labels:
+        team: infra
+        severity: warning
+      data:
+      - refId: A
+        queryType: ''
+        relativeTimeRange:
+          from: 600
+          to: 0
+        datasourceUid: PD8C576611E62080A
```

Comment:
Odd question - when you provision a dashboard, how does it handle datasourceUID? Presumably datasourceUIDs will be different on different instances? How does this not break, does it have a system for pointing to another resource?

Reply:
If dashboards has a mechanism for this, let's lift it

Reply:
Talked about this offline. UID should be user provided.